### DISCUSSION TITLE
KeepassXC Multiuser Logins

### DISCUSSION THREAD

**Discussion by BlackMontana12** [keepassxreboot/keepassxc#8160] [Q&A]
Hi there, 
Here is the case:
I would like to create a Database for some passwords that we share with my coworkers. I downloaded KeepassXC and it works great. The only Problem I have is that I could not find where to create multiple Logins for my Coworkers, because i don't want to give everybody the Masterpassword. Is there a Solution how to make these Logins? Or is there a similar Password Manager on the market?

Thanks for the replies :)

**Comment by droidmonkey** [comment]
Use keeshare to create a shared database and drop credentials in there that you want to share. Read the user guide for more info.

**Reply by BlackMontana12** [reply]
Thanks a lot Keeshare was the word I was searching :) I will read up on it.
